### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@cfconrad WDYT ?

File: isos_post.py [hunk 1]
```diff
@@ -82,9 +85,9 @@ def set_job_priority(host, job_id, priority):
                             'wicked_startandstop': 'wicked_startandstop_ref,wicked_startandstop_sut',
                             'wicked_aggregate': 'wicked_aggregate_ref,wicked_aggregate_sut'}
     if (args.alias == 'list'):
-        print('Aliases:')
+        logging.debug('Aliases:')
```
Comment (cfconrad) [inline isos_post.py]: Not sure if we should use `logging` here. Doesn't have a logger a time prefix and some other information which we do not need here?

**Review by cfconrad — COMMENTED** [review]
In general the right direction. I wonder that I don't find a ` setLevel(level)` call.

**Comment by asmorodskyi** [comment]
> In general the right direction. I wonder that I don't find a ` setLevel(level)` call.

yeah makes sense , added

File: isos_post.py (hunk 1 above)
Comment (asmorodskyi) [reply isos_post.py]: I added special formatting as for now it is ok

**Pull request merged**